src/backend/app.js

```diff
@@ -21,10 +23,53 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+const db = Knex({
```

Comment:
The database should be connected through a .env file as this info contains sensitive information like username and password. This is the crucial thing that a developer should be aware of.

Reply:
Hi, Thank you so much for your feedback.
I have fixed the code to connect through .env now :)